provider/google/google.go

```diff
@@ -102,7 +102,7 @@ type GoogleProvider struct {
 	provider.BaseProvider
 	// The Google project to work in
 	project string
-	// Enabled dry-run will print any modifying actions rather than execute them.
+	// IsEnabled dry-run will print any modifying actions rather than execute them.
```

Comment:
It looks an unexpected change or did I miss something ?

Reply:
fixed